net/cloudflared/files/sample_config.yml

```
@@ -0,0 +1,3 @@
+url: http://localhost:80
```

```suggestion
url: http://localhost:8000
```

To match Cloudflare's documentation, and to not accidentally tunnel luci.

---

Thank you for your suggestion!